test/unit/matrix/test_matrix_output.cpp

```diff
@@ -206,7 +272,10 @@ struct test_matrix_dist_output {
 
     const std::string output{"M = np.zeros((0, 0), dtype=np.single)\n"};
 
-    return std::make_pair(std::move(mat), output);
+    std::stringstream output_csv;
+    output_csv << mat << std::endl;
```

Comment:
Here I would specify what's the expected output, without relying on code for producing it.

Reply:
I don't agree.
As discussed before (unfortunately it is not documented in the code), if the matrix is distributed the basic info of the matrix are printed using the operator<<.
This is the expected behaviour, independently from the implementation of matrix operator<<.